src/lib/tasks/Render.ts

```diff
@@ -27,17 +28,57 @@ export class RenderTask extends Task<RenderTaskParams> {
     }
     this.metrics.goto = Date.now() - start;
 
-    /* Transforming */
     const statusCode = response.status();
-    const baseHref = `${url.protocol}//${url.host}`;
-    await page!.evaluate(injectBaseHref, baseHref);
+    const headers = response.headers();
 
     start = Date.now();
     if (statusCode === 200 && minWait) {
       await page!.waitForTimeout(minWait - (Date.now() - total));
     }
     this.metrics.minWait = Date.now() - start;
 
+    if (page!.url() !== url.href) {
+      this.results = {
+        statusCode,
+        headers,
+        resolvedUrl: page!.url(),
+      };
+      return;
+    }
+
+    try {
+      const metaRefreshElement = await page!.$('meta[http-equiv="refresh"]');
+      if (metaRefreshElement) {
+        console.log(await metaRefreshElement.jsonValue());
+        const metaRefreshContent = await metaRefreshElement.getProperty(
+          'content'
+        );
+        const refreshContent = await metaRefreshContent?.jsonValue<string>();
+        const match = refreshContent?.match(/\d+;\s(?:url|URL)=(.*)/);
```

Comment:
regex could be a constant +
```suggestion
        const match = refreshContent.match(/\d+;\s(?:url|URL)=(.*)/);
```

Reply:
`TS2532: Object is possibly 'undefined'.`